lua/lvim/core/treesitter.lua

```diff
@@ -89,6 +124,13 @@ M.setup = function()
 
   treesitter_configs.setup(opts)
 
+  if vim.opt.foldmethod:get() == "expr" and vim.opt.foldexpr:get() == "nvim_treesitter#foldexpr()" then
+    vim.opt.foldmethod = "manual"
+  end
```

Comment:
what's the reason for overriding this here? it's not like we're setting these values anyway

Reply:
I had treesitter folds in my config and they slowed down opening of big files a lot but since it's not the default in lunarvim I will remove it from here

Reply:
```suggestion
```